virtcontainers/sandbox.go

```diff
@@ -1404,7 +1410,30 @@ func (s *Sandbox) UpdateContainer(containerID string, resources specs.LinuxResou
 		return err
 	}
 
-	return c.update(resources)
+	newResources := ContainerResources{
+		VCPUs: uint32(utils.ConstraintsToVCPUs(*resources.CPU.Quota, *resources.CPU.Period)),
```

Comment:
You are effectively mapping to vCPU at the container level, here, which I do not think we want.

We should calculate in terms of CPU shares/quota at the sandbox level, and only after an adjustment is made should we translate to vCPU.  That way two containers asking for 0.2 CPU each will result in 1vCPU for, not 2.